### ISSUE TITLE
a570 problem

### ISSUE THREAD

**Issue by CCKMik** [OlegMishin/A500_8MB_ide2#8]
Current firmware does not support a570 cdrom, red screen, no 8mb ram.

Is this caused by DMAC or a570/cdtv ext boot rom?

Will there ever be firmware that supports a570/cdtv ??

**Comment by OlegMishin** [comment]
That's right. CDTV is not supported due to DMAC. I guess same issue with A570. My device has to be the only Autocofig unit in system. It can be fixed by firmware by skipping the first Autoconfig request, but I don't have neither CDTV nor A570 to test.
Both devices are quite expencive though :)
